### PR TITLE
Update deps

### PR DESCRIPTION
**Pull request by tomasklim**
## Description

Most of the libraries do not have any breaking changes.

Notes:
- `@types/node` We forgot to update this package when updating to v16 #5585
- I updated some of the storybook files as we are using deprecated lib `@storybook/addon-knobs` #3907, there are still some left

After merging, I will continue with updating Babel and TypeScript.

### CODE REVIEW

File: packages/connect/setupJest.ts [hunk 1]
```diff
@@ -70,22 +70,19 @@ const getReleasesT1 = (releases: Partial<FirmwareRelease>[]) => releases.map(r =
...
-                getReleasesT2: typeof getReleasesT2;
-            };
-        }
-        interface ProcessEnv {
-            RELEASES_T1: string;
-            RELEASES_T2: string;
-            BASE_FW_URL: string;
-        }
+    // eslint-disable-next-line no-var, vars-on-top
+    var JestMocks: {
```
Comment (dahaca) [inline packages/connect/setupJest.ts:74]: `var`?

File: scripts/package.json [hunk 2]
```diff
@@ -10,11 +10,7 @@
         "chalk": "^4.1.2",
         "fs-extra": "^10.1.0",
         "minimatch": "^3.0.4",
-        "prettier": "2.5.1",
         "sort-package-json": "^1.54.0",
         "yargs": "17.5.1"
-    },
-    "devDependencies": {
-        "@types/prettier": "2.4.3"
```
Comment (dahaca) [inline scripts/package.json:18]: Why?

Comment (tomasklim) [reply scripts/package.json:18]: It is in root package.json

File: packages/connect/setupJest.ts (hunk 1 above)
Comment (tomasklim) [reply packages/connect/setupJest.ts:74]: Does not work with const or let

File: scripts/package.json (hunk 2 above)
Comment (tomasklim) [reply scripts/package.json:18]: reverted

File: packages/connect/setupJest.ts (hunk 1 above)
Comment (dahaca) [reply packages/connect/setupJest.ts:74]: Pardon, read about global variables in TS now, got it :) Was a bit confused with the `interface` => `var` change

**Review by dahaca — APPROVED** [review]

**Pull request merged**